src/Main.java

```diff
@@ -1,5 +1,64 @@
+import java.time.LocalDate;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        int deliveruDisnance = (int) 101f;
+        boolean deliveruDisnanceMax = deliveruDisnance <= 100;
+        PrintDeliveruDisnance(deliveruDisnanceMax, deliveruDisnance);
+
+
+        System.out.println();
+        int clientOS = 0;
+        int clientDeviceYear = 2015;
+        recommedAppVersion(clientOS, clientDeviceYear);
+
+        System.out.println("1");
+
+
+        printIsLeapYearResult(2020);
+        printIsLeapYearResult(2021);
+        printIsLeapYearResult(2022);
+        printIsLeapYearResult(2023);
+        printIsLeapYearResult(2024);
     }
-}
\ No newline at end of file
+
+    private static void PrintDeliveruDisnance(boolean deliveruDisnanceMax, int deliveruDisnance) {
```

Comment:
```suggestion
    private static void printDeliveruDisnance(boolean deliveruDisnanceMax, int deliveruDisnance) {
```
методы надо называть с маленькой буквы

Reply:
метод по заданию должен принимать 1 параметр и должен возвращать результат, т.е. выводить на печать ничего не должен, не его это задача. Его задача просто посчитать кол-во дней и вернуть, а вот кто его вызовет - тот уже может вывести на печать результат